internal/discovery/services_lister.go

```diff
@@ -0,0 +1,25 @@
+package discovery
+
+import (
+	"time"
+
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	v1 "k8s.io/client-go/listers/core/v1"
```

Comment:
We named this import `listersv1` in the other package. I don't mind either but we probably should choose only one.